devops/README.md

```diff
@@ -0,0 +1,46 @@
+# CommIT Smart
+
+[commitsmart.com](https://commitsmart.com)
+
+[info@commitsmart.com](mailto:info@commitsmart.com)
+
+## DevOps Assignment
+
+The goal of the assignment is to get a picture of your programming skills and coding style.
+
+Don't stress if you can't get every part done -- spend a maximum of 3 hours on this assignment. Feel free to add comments or README notes to help us understand your thinking and decisions.
+
+Submit your code through `github.com` by creating a **private** repository and sharing it with the `csabaujvari` github account.
+
+---
+
+## Assignment
+
+1. Create a Terraform configuration that provisions a GCP Kubernetes Engine (GKE) cluster, including the necessary resources such as the cluster, nodes, and firewall rules.
+1. Use Terraform to provision a GCP Persistent Disk and attach it to the cluster.
+1. Use Terraform to provision a GCP Cloud SQL instance and create a database on it.
+1. Use Terraform to provision a GCP Cloud Storage bucket and upload a file to it.
+1. Use Git for version control and GitHub for code sharing.
+1. Create a Kubernetes deployment and service configuration that deploys a simple web application on the cluster and exposes it to the internet.
+1. Create a detailed document that explains the architecture of the infrastructure, the technologies used, and the reasoning behind the design decisions made.
+1. Create a script that automates the process of applying the Terraform configuration and running the test (if you have).
```

Comment:
Create a CI/CD pipeline for deploying the terraform code and the k8s manifests? Or just one of them? with Jenkins or gitlab-ci